apidoc/Titanium/UI/Window.yml

```diff
@@ -439,6 +439,28 @@ properties:
     since: 3.1.3
     osver: {ios: {min: "7.0"}}
 
+  - name: extendSafeArea
+    summary: |
+       Specifies whether the content (subviews) of the window  will render inside the safe-area or not. 
+       Only used in iOS 11 and later.
+    description:  |
+        If the value is `true`, the content (subviews) of the window will not respect the safe-area, meaning that the content 
+        will fill the whole screen available for rendering. If the value is `false`, the content of the
+        window will be rendered inside the safe-area (including then status-bar, iPhone X home-indicator & iPhone X notch).
```

Comment:
**Question:**
If this is set `false`, then what will this mean for a translucent status bar? Will the window's background color or image be behind the translucent status bar or not? If yes, then does the window's `height` property include the height of the status bar then?

I ask because we may want to document these details. This may matter to a developer who is doing absolute positioning via a composite layout that involves subtracting from the window height.  Please see the "LayoutTest.js" sample project attached to [TIMOB-24972](https://jira.appcelerator.org/browse/TIMOB-24792) and try its "Absolute Position" test in the list.

Reply:
If `false`, it will not extend the safe-area (status-bar is part of that), meaning that a translucent status-bar should receive the window-background-color and the height of the window will not be influenced by the status-bar.

Reply:
Okay, so, this means that `window.height` and `window.width` returns the safe area width height in this case, right? We should document this detail.